apps/showcase/src/app/app.module.ts

```diff
@@ -40,12 +41,15 @@ const runtimeChecks: Partial<RuntimeChecks> = {
 registerLocaleData(localeEN, 'en-GB');
 registerLocaleData(localeFR, 'fr-FR');
 
+const logger = new ConsoleLogger();
```

Comment:
Should not be `LoggerService` injected by Angular instead of being manually instantiated ?

Reply:
LoggerService is provided in @o3r/logger which depends on @o3r/core. Do we really want to add this dependency?